pynamodb/models.pyi

```diff
@@ -29,7 +29,7 @@ class Model(metaclass=MetaModel):
     _hash_keyname: Optional[str]
     _range_keyname: Optional[str]
     _connection: Optional[TableConnection]
-    def __init__(self, hash_key: Optional[KeyType] = ..., range_key: Optional[Any] = ..., **attrs) -> None: ...
+    def __init__(self, hash_key: Optional[KeyType] = ..., range_key: Optional[Any] = ..., _user_instantiated: bool, **attrs) -> None: ...
```

Comment:
@irhkang since it's an internal parameter, maybe it **shouldn't** be part of the typeshed?